src/styles/_calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
```

Comment:
They ask us to be generous and to add extra padding so the calendar day box doesnt go outside our screen you can make it 25px or 30 px. It would look nicer.